.github/workflows/pythonapp.yml

```diff
@@ -16,28 +16,42 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
@ldruschk should we nuke some of the older python versions while we are at it?

Reply:
Pytohn 3.8 is technically still not EOL and afaik there are no major features in newer versions that we would want to use in this package